package.json

```diff
@@ -33,7 +33,8 @@
     "http": "static spec/jasmine -H '{\"Cache-Control\": \"no-cache, must-revalidate\"}'",
     "****** NODE *****": "",
     "postinstall": "cd docs && yarn",
-    "precommit": "lint-staged & grunt exec:docs_dist & wait && git add -A docs/{index.html,dist/**/*}"
+    "precommit": "lint-staged & grunt exec:docs_dist",
```

Comment:
also, we should probably add `docs/src/less/ts-runtime.less` to git as well because it's generated programmatically now.

```
"precommit": "lint-staged && grunt exec:docs_dist && git add -A docs/{index.html,src/less/ts-runtime.less,dist/**/*}"
```